docs/UserGuide.md

```diff
@@ -744,54 +791,89 @@ Important notes:
 > Categories: #shopping
 > ```
 > Input:
-> <img src="images/unicash/command-outputs/getTransaction/getInitial2.png" width="1000" />
+> <img src="images/unicash/command-outputs/getTransaction/GetInitial3.png" width="1000" />
 > Ouput:
-> <img src="images/unicash/command-outputs/getTransaction/getSuccessFinal2.png" width="1000" />
+> <img src="images/unicash/command-outputs/getTransaction/GetSuccess.png" width="1000" />
 
 ##### Failed Execution
 
 ###### Example 1
 
-> **Case**: Missing compulsory fields.
+> **Case**: Missing compulsory fields i.e. `INDEX`
 >
 > **Input**: `get`
 >
 > **Output**:
 > ```
 > Invalid command format! 
 >
-> get: Displays expanded details of a specific transaction.
+> get, g: Displays expanded details of a specific transaction.
 >
 > Argument: Index (must be a positive integer)
 >
 > Example: get 2
 > ```
-> <img src="images/unicash/command-outputs/getTransaction/getFail1.png" width="1000" />
 
 ###### Example 2
 
-> **Case**: Invalid `<INDEX>` provided
+> **Case**: Invalid `INDEX` provided
 >
-> **Context**: `10` given as `<INDEX>` when only `5` transactions are presently displayed.
+> **Context**:`INDEX` given as `10` when only `5` transactions are present
+> in the current `Transaction List` configuration
 >
 > **Input**: `get 10`
 >
 > **Output**:
 > ```
 > The transaction index provided is invalid
 > ```
-> <img src="images/unicash/command-outputs/getTransaction/getFail2.png" width="1000" />
+
+###### Example 3
+
+> **Case**: Invalid `INDEX` provided
+>
+> **Context**: `INDEX` given as a negative number
+>
+> **Inputs**: `get -1`
+>
+> **Output**:
+> ```
+> Invalid command format! 
+>
+> get, g: Displays expanded details of a specific transaction.
+>
+> Argument: Index (must be a positive integer)
+>
+> Example: get 2
+> ```
+
+###### Example 4
+
+> **Case**: Invalid `INDEX` provided
+>
+> **Context**: `INDEX` given as a number larger than `Integer.MAX_VALUE`
+>
+> **Inputs**: `get 10000000000000000`
+>
+> **Output**:
+> ```
+> Invalid command format! 
+>
+> get, g: Displays expanded details of a specific transaction.
+>
+> Argument: Index (must be a positive integer)
+>
+> Example: get 2
+> ```
+
 
 #### Find Transaction(s)
 
 Finds a `Transaction` in UniCa$h.
 
 Command: `find [n/NAME] [l/LOCATION] [c/CATEGORY]`
 
-Command Words Accepted: `find`, `search`, `f` (case-insensitive)
-
-Command Argument: `<INDEX>` is the displayed transaction index
-of the transaction to be retrieved, as shown in the `Transactions List`.
+Command Words Accepted: `find`, `search`, `f` (all case-insensitive)
 
 Command Options:
```

Comment:
Good to use either `Parameters` or `Prefixes` instead of `Options` to maintain consistency with the above documentation

Reply:
Also good to link to the prefix types for value specific constraints

Reply:
Done, changed to Parameters and also added the link to the prefix constraints